hack/release.toml

```diff
@@ -6,9 +6,9 @@ github_repo = "talos-systems/sidero"
 match_deps = "^github.com/(talos-systems/[a-zA-Z0-9-]+)$"
 
 # previous release
-previous = "v0.4.0"
+previous = "v0.5.0"
 
-pre_release = true
```

Comment:
I think we should clean up release notes here from items which were already in 0.5.0

Reply:
so it lists only _new_ items

Reply:
so no notes, that's fine as well :)

Reply:
I've added some :)